client/blocks/eligibility-warnings/index.tsx

```diff
@@ -140,22 +151,29 @@ export const EligibilityWarnings = ( {
 					/>
 				</CompactCard>
 			) }
-			{ ( title || primaryText ) && (
+			{ ! isPlaceholder && context === 'plugin-details' && (
```

Comment:
This should probably go in the holds component now that I see it inlined. don't think it's necessary for merge though.